packages/@n8n/nodes-langchain/nodes/vector_store/shared/methods/listSearch.ts

```diff
@@ -67,3 +68,25 @@ export async function qdrantCollectionsSearch(this: ILoadOptionsFunctions) {
 
 	return { results };
 }
+
+export async function mongoCollectionSearch(this: ILoadOptionsFunctions) {
+	const credentials = await this.getCredentials('mongoDb');
+
+	const client = new MongoClient(credentials.connectionString as string, {
+		appName: 'devrel.content.n8n_vector_integ',
```

Comment:
is this `appName` needed? if yes, does it make sense to let the users customize this?

Reply:
Its a way for use to track connection origin to messure impact of the integration on the Atlas Side. It should be set to this value.